src/containers/WindowSideBarAnnotationsPanel.js

```diff
@@ -28,6 +29,7 @@ function getIdAndContentOfResources(resources) {
  * @private
  */
 const mapStateToProps = (state, { windowId }) => ({
+  allAnnotationsAreHighlighted: getWindow(state, { windowId }).displayAllAnnotations,
```

Comment:
I _could_ change this state/config attribute name do `highlightAllAnnotations`.  Not sure if it matters.